src/components/ApartmentApplicantList/components/ApplicantListItem/index.js

```diff
@@ -0,0 +1,195 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import 'date-fns';
+import {
+  Grid,
+  ListItem,
+  ListItemAvatar,
+  ListItemSecondaryAction,
+  ListItemText,
+  Avatar,
+  Button,
+  Menu,
+  MenuItem,
+  ListItemIcon,
+} from '@material-ui/core';
+import PersonIcon from '@material-ui/icons/Person';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import StarBorderIcon from '@material-ui/icons/StarBorder';
+import ClearIcon from '@material-ui/icons/Clear';
+import user from '../../../../services/user';
+import '../../../../views/ApartmentApp/apartmentApp.css';
+import '../../../../views/PeopleSearch/components/PeopleSearchResult/peopleSearchResult.css';
+
+// Based off src/views/PeopleSearch/components/PeopleSearchResult
+// but using this.props.profile of type StudentProfileInfo
+// rather than using this.props.Person of type PeopleSearchResult
+export default class ApplicantListItem extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      avatar: null,
+      anchorEl: null, // A HTML element, or a function that returns it. It's used to set the position of the menu.
+    };
+  }
+
+  componentDidUpdate(newProps) {
+    if (this.props.profile.AD_Username !== newProps.profile.AD_Username) {
+      this.loadAvatar();
+    }
+  }
+
+  componentDidMount() {
+    this.loadAvatar();
+  }
+
+  async loadAvatar() {
+    this.setState({ avatar: null });
+    const [{ def: defaultImage, pref: preferredImage }] = await Promise.all([
+      await user.getImage(this.props.profile.AD_Username),
+    ]);
+    let avatar;
+    if (this.props.profile.AD_Username) {
+      avatar = preferredImage || defaultImage;
+    } else {
+      avatar = (
+        <svg width="50" height="50" viewBox="0 0 50 50">
+          <rect width="50" height="50" rx="10" ry="10" fill="#CCC" />
+        </svg>
+      );
+    }
+    this.setState({ avatar });
+  }
+
+  handleMenuClick = (event) => {
+    this.setState({ anchorEl: event.currentTarget });
+  };
+
+  handleMenuClose = () => {
+    this.setState({ anchorEl: null });
+  };
+
+  handleChangePrimary = (profile) => {
+    // Make sure the chosen profile was not null
+    if (profile) {
+      // Send the selected profile to the parent component
+      this.props.onChangePrimary(profile);
+      this.handleMenuClose();
+    }
+  };
+
+  handleRemove = (profile) => {
+    // Make sure the chosen profile was not null
+    if (profile) {
+      // Send the selected profile to the parent component
+      this.props.onApplicantRemove(profile);
+      this.handleMenuClose();
```

Comment:
I think that the menu would still be closed by other actions such as clicking outside the menu, but that menu has since been removed so it is no longer needed